Flow.Launcher/Converters/IconRadiusConverter.cs

```diff
@@ -8,15 +8,10 @@ public class IconRadiusConverter : IMultiValueConverter
     {
         public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
-            if (values.Length != 2)
-                throw new ArgumentException("IconRadiusConverter must have 2 parameters");
+            if (values is not [double size, bool half])
```

Comment:
Original code didn't have the variable names, I didn't know how to call them, so I jsut called them `size` and `half`.

Reply:
I think `half` is not the correct variable name. The reason it is divided by 2 is that if not radius will be large enough so that the icon didn't get cut. Maybe something like `enableCircleIcon` or something like that might be better. I am also terrible at naming🤣